.ci/no_old_refs.sh

```diff
@@ -3,29 +3,76 @@
 set -e
 
 # Script requires GITHUB_TOKEN env variable
-MENTIONED_ISSUES=/tmp/mentioned_issues
+MENTIONED_ISSUES_GREP_OUTPUT=/tmp/mentioned_issues_grep_output
 CLOSED_ISSUES=/tmp/failed_issues
+LINKED_ISSUES_FORMATTED=/tmp/linked_issues
+CHECKSTYLE_ISSUE_URL="https:\/\/github.com\/checkstyle\/checkstyle\/issues\/"
+
+# $LINKED_ISSUES need formatting before the are used
+if [ ! -z "$LINKED_ISSUES" ]; then
```

Comment:
I tihnk it is better to have this formatting in pipeline step where we get these links.
Also, it will be good to show input/output example or description what "formatting" means

Reply:
I added this as a step in the workflow, also added an explanatory comment.